crutest/src/main.rs

```diff
@@ -1053,6 +1107,17 @@ async fn main() -> Result<()> {
                 return Ok(());
             }
         }
+        Workload::Bufferbloat { cfg } => {
```

Comment:
The workloads were in alphabetical order.  Maybe put this
one up between `Workload::Biggest` and `Workload::Burst`?